src/main/java/htsjdk/samtools/SAMSequenceDictionary.java

```diff
@@ -175,6 +175,33 @@ public void assertSameDictionary(final SAMSequenceDictionary that) {
             throw new AssertionError(String.format(DICT_MISMATCH_TEMPLATE, thatSequences.next() + " is present in only one dictionary"));
     }
 
+    /**
+     * Non-comprehensive {@link #equals(Object)}-assertion: instead of calling {@link SAMSequenceRecord#equals(Object)} on constituent
+     * {@link SAMSequenceRecord}s in this dictionary against its pair in the target dictionary, in order,  call
+     * {@link SAMSequenceRecord#isSameSequence(SAMSequenceRecord)}.
```

Comment:
No. The comment is for each `SAMSequenceRecord`. `isSameSequence` is a less strict version of `equals`. Changed `assertion` to `validation`.